build/uploadperftobenchview.sh

```diff
@@ -63,6 +63,8 @@ else
     GIT_BRANCH_WITHOUT_ORIGIN=$GIT_BRANCH
 fi
 
+$timeStamp = date +'%Y-%m-%dT%H:%M:%SZ'
```

Comment:
This doesn't look right:

```
> $timeStamp = date +'%Y-%m-%dT%H:%M:%SZ'
=: command not found
```


I would expect something like:
``` 
timeStamp=$(date +'%Y-%m-%dT%H:%M:%SZ')
```

Reply:
This is the form:
timeStamp=`date +%Y-%m-%dT%H:%M:%SZ`
the trick is the back-ticks